storage/innobase/include/row0upd.h

```diff
@@ -577,6 +582,9 @@ struct upd_node_t{
 				/* column assignment list */
 	ulint		magic_n;
 
+	/** System Versioning: modify update vector to set row_start
+	 * (or row_end in case of DELETE) to current trx_id. */
+	void		vers_set_fields(const trx_t* trx);
```

Comment:
When and how exactly is the update vector modified? Are new fields added? Do not add `*` in the middle of Doxygen comments. Replace `trx_id` with `trx->id`. Maybe the function comment should refer to `field->col->vers_native()` or `dict_col_t::vers_native()`?